leaderboardsgg/leaderboard-site
# Upgraded ESLint to 8 and various Dev Experience upgrades/ fixes

## What

Upgraded ESLint to 8 and fixed warnings/ errors found from such, updated some developer experience related files to not show errors, and added lang=postcss to all style tags in vue components so that there are no longer warnings and errors and allows for actual linting of the tailwind that is written.

### Steps for testing

1. Checkout to branch
2. run `pnpm install`
3. Open in VSCode and ensure you have the extensions listed in `extensions.json` all installed
4. close and reopen VSCode to be sure
5. open various `.vue`, `.ts`, etc files and ensure there are no warnings/ errors
